### ISSUE TITLE
QUESTION: Workflow question - how to archive done tasks

### ISSUE THREAD

**Issue by tott** [nvim-orgmode/orgmode#413]
I love the plugin but wondering a bit in terms of workflow efficiency.

When I work on my tasks I usually do my work and then mark the completed tasks as DONE at some point. At this state I usually still want to keep them in the list until I get full confirmation which then clutters my list a bit and I am looking for a way to somehow change the state of all tasks with a certain state. ie. "archive all done tasks" or something like this.

This feels fairly common to me but I'm wondering if I'm just generally approaching this wrong and would be interested to hear what the intended workflow would be or how I could achieve alterations for a group of tasks by state or tag?

**Comment by jgollenz** [comment]
Do you mean all done tasks in the whole buffer or only the ones under the current heading?

**Comment by tott** [comment]
Yes, I usually would do a task review a few times a day and update tasks that are moving through the pipe so it would be great after i do my review to just archive all the done tasks for example.

**Comment by jgollenz** [comment]
If the workflow described [here](https://stackoverflow.com/a/6998051) would work in this port, do you think it would suit your needs? It actually sounds like something that in the future could be done with the API as well. Can't find anything in Emacs orgmode that does exactly what you describe.

**Comment by tott** [comment]
I tried this but can't find a way to filter by status or show all. It's either showing TODO or lets me filter by tags only

**Comment by zeylos** [comment]
Sorry for diging an old issue but I have the same question, the easiest way I found is to use [Advanced search](https://github.com/nvim-orgmode/orgmode/blob/master/DOCS.md#advanced-search) with something like `/DONE` and use the archive shortcut on every line, which should be easy to automate but I can't find how, someone has a hint ?

btw it looks like org-mode has it in core https://github.com/bzg/org-mode/blob/main/lisp/org-archive.el#L506

**Comment by ficcdaf** [comment]
I'd be interested in reviving this issue and implementing this feature. my only reservation is that I've never used emacs so I don't have a good frame of reference for if something behaves exactly how it does in emacs.
